clang/test/Driver/basic-block-address-map.c

```diff
@@ -1,4 +1,5 @@
 // RUN: %clang -### -target x86_64 -fbasic-block-address-map %s -S 2>&1 | FileCheck -check-prefix=CHECK-PRESENT %s
+// RUN: %clang -### -target aarch64 -fbasic-block-address-map %s -S 2>&1 | FileCheck -check-prefix=CHECK-PRESENT %s
```

Comment:
`--target=` for new tests. `-target ` has been deprecated since ~ 3.4

Reply:
Done!